Add another variant of command-line tag to expressions to be removed

Signed-off-by: Jonas Karlsson <jonas@gobolinux.org>

--- perl-5.8.8/makedepend.SH	2003-06-05 18:11:10.000000000 +0000
+++ perl-5.8.8/makedepend.SH.new	2008-04-09 20:28:10.000000000 +0000
@@ -166,6 +166,7 @@
 	    -e '/^#.*<stdin>/d' \
             -e '/^#.*<builtin>/d' \
             -e '/^#.*<built-in>/d' \
+            -e '/^#.*<command-line>/d' \
             -e '/^#.*<command line>/d' \
 	    -e '/^#.*"-"/d' \
 	    -e '/: file path prefix .* never used$/d' \
